.github/workflows/nodejs.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - uses: actions/setup-node@v3
       - run: npm install --legacy-peer-deps
       - run: npm run lint
+      - run: npm run toc
```

Comment:
Out of curiosity, would it fail if any modification happens?

Reply:
Good point; no, it won't. I'd somehow gotten it into my head that a dirty worktree would cause a failure somewhere in this task, but clearly that's not the case. Will fix.